databaseConnection.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+function connectDB(){
+    mongoose.connect('mongodb://localhost/testDb',{
```

Comment:
The database URL has to be in the .env file since it will be different on other platforms

Reply:
Moved databse url to .env file